dist/cookie/{{cookiecutter.project_slug}}/modules/utils.js

```diff
@@ -78,3 +79,13 @@ export function getPropertyMap(source, prop) {
   });
   return map;
 }
+
+export function getNavigationScreen(name) {
+  const screen = screens.find(x => x[0].includes(name))
+
+  if(screen){
+    return screen[0]
+  }
+
+  return name
+}
```

Comment:
This function should be implemented in `@screens` directly:
https://github.com/crowdbotics/modules/blob/55694487cd48f64c5f280924ef3efd9dd3242d7d/scaffold/template/custom/screens/index.js#L3

And then on the figma screens generated code you can replace this:
```
import { getNavigationScreen } from "../../modules/utils"
```

With a non relative import, that will always work:
```
import { getNavigationScreen } from "@screens"
```